pkg/operator/handlers/dapr_handler.go

```diff
@@ -78,11 +79,13 @@ func (h *DaprHandler) Init() error {
 				return nil
 			}
 			return []string{owner.Name}
-		}); err != nil {
+		},
+	)
+	if err != nil {
```

Comment:
why is the short if changed to this format?

Reply:
IMHO a function inside an if was unreadable :)